mod-mosaic/schemas/mosaic_order_request.json

```diff
@@ -0,0 +1,150 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Mosaic Configuration",
+  "type": "object",
+  "javaName": "MosaicConfiguration",
+  "properties": {
+    "id": {
+      "description": "UUID of the configuration",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "title": {
+      "description": "title of the material",
+      "type": "string"
+    },
+    "author": {
+      "description": "the name of a author to the material",
+      "type": "string"
+    },
+    "publicationDate": {
+      "description": "date (year) of the material's publication",
+      "type": "string"
+    },
+    "edition": {
+      "description": "edition of the material",
+      "type": "string"
+    },
+    "productId": {
+      "description": "The actual product identifier",
+      "type": "string"
+    },
+    "receivingNote": {
+      "description": "notes regarding receiving instructions",
+      "type": "string"
+    },
+    "userLimit": {
+      "description": "the concurrent user-limit",
+      "type": "string"
+    },
+    "requesterName": {
+      "description": "who requested this purchase order line",
+      "type": "string"
+    },
+    "selectorName": {
+      "description": "who selected this material",
+      "type": "string"
+    },
+    "poLineNote": {
+      "description": "note for poLIne",
+      "type": "string"
+    },
+    "poLineDescriptor": {
+      "description": "descriptor for poLine",
+      "type": "string"
+    },
+    "renewalNote": {
+      "description": "Renewal note for this purchase order line",
+      "type": "string"
+    },
+    "referenceNumbers": {
+      "description": "Collection of reference number items",
+      "$ref": "../../common/schemas/reference_numbers.json"
+    },
+    "listUnitPrice": {
+      "description": "The per-item list price for physical or resources of 'Other' order format",
+      "type": "number"
+    },
+    "listUnitPriceElectronic": {
+      "description": "The e-resource per-item list price",
+      "type": "number"
+    },
+    "currency": {
+      "description": "An ISO currency code",
+      "type": "string"
+    },
+    "quantityPhysical": {
+      "description": "Quantity of physical items or resources of 'Other' order format in this purchase order line",
+      "type": "integer"
+    },
+    "quantityElectronic": {
+      "description": "Quantity of electronic items in this purchase order line",
+      "type": "integer"
+    },
+    "folioOrderTemplateId": {
```

Comment:
Let's move this property at higher level, I think it should be something like that
mosaic_order_request
  orderTemplateId: string
  orderData: object

Reply:
Divided to mosaic order and mosaic order request to include these two strucutre